tests/integration/netdev.bats

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/env bats
+
+load helpers
+
+function setup() {
+	setup_busybox
+}
+
+function teardown() {
+	teardown_bundle
+}
+
+@test "move network device to container network namespace" {
+	requires root
+	# create a dummy interface to move to the container
+	ip link add dummy0 type dummy
```

Comment:
Maybe these network device setup operations need to be moved to `setup()`?
And the network device removal operations need to be moved to `teardown()`.

Reply:
not familiar with the test framework, the network device setup indeed is not part of the test, but the network removal asserts that the device is returned to the original namespace instead of being destroyed, so I think is better to keep it within the test body

Reply:
Once a assert in the test is failed, the test will be ended, so the rest codes after this assert will not be executed. It will cause the network device we added before remaining in the host.

Reply:
good point, I will move them accordenly

Reply:
done